README.rst

```diff
@@ -139,14 +139,14 @@ mailmojo\_auth
 -  **Authorization URL**: https://api.mailmojo.no/oauth/authorize/
 -  **Scopes**:
 
-  -  **account**: Manage your MailMojo account.
```

Comment:
This "de-indent" of scopes list also does not seem correct. The list is a sub-list of the **Scopes** list item.